maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -59,9 +92,12 @@ class IOSDriver(
 
     override fun close() {
         iosDevice.close()
+        iosDevice.uninstall(IOSUiTestRunner.UI_TEST_RUNNER_APP_BUNDLE_ID)
```

Comment:
I think we need to uninstall it before installing it again. There is no guarantee that `close()` will always be called

Reply:
Done